bower.json

```diff
@@ -5,10 +5,16 @@
     "dist/rollbar.umd.js"
   ],
   "ignore": [
+    "dist/*.snippet*",
+    "dist/*.nojson*",
+    "dist/*.named-amd*",
```

Comment:
Thanks for the ping @sandstrom. I missed it the first time.

The `named-amd` build target is still created. This only removes it from the bower distribution since the UMD target is listed as the `main` JS file.